index.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css"> 
+    <title>Capstone Web Page</title>
+</head>
+
+
+<body>
+    <header id="navbar">
+        <nav>
+            <div id="log-section">
+                <a href="#" id="my-page">My page</a>
+                <a href="#" id="log-in-out">Log in</a>
+            </div>
```

Comment:
- Make sure the top short bar section with social media links, logout, and language links aligned to the right.
- **See Original design**:
![Capture](https://user-images.githubusercontent.com/63972670/210216783-fc4b78ef-276d-4c95-8e8d-46591348d686.PNG)